artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml

```diff
@@ -106,11 +107,8 @@ spec:
           - description: 'Informs the status of vSphere Storage Provider'
             displayName: 'Storage Provider'
             path: csi
-  description: 'Vsphere Kubernetes Drivers Operator helps to install/manage vsphere drivers required to run kubernetes workload on vSphere. The operator currently manages CPI and CSI vSphere drivers. The operator exposes two resources: VsphereCloudConfig - represents the connection information required to connect to vcenter. VDOConfig - represents the configuration information w.r.t CPI and CSI'
-  displayName: Vsphere Kubernetes Drivers Operator
-  icon:
-    - base64data: PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA0MDAgNDAwIj48cGF0aCBkPSJNMzMxLjkyMiwwSDIyNC4wNDRsLTE2LDE2SDMzMS45MjJBNTIuMTM3LDUyLjEzNywwLDAsMSwzODQsNjguMDc4VjMzMS45MjJBNTIuMTM3LDUyLjEzNywwLDAsMSwzMzEuOTIyLDM4NEgyMjcuMzM5bC0xNiwxNkgzMzEuOTIyQTY4LjE1Niw2OC4xNTYsMCwwLDAsNDAwLDMzMS45MjJWNjguMDc4QTY4LjE1Niw2OC4xNTYsMCwwLDAsMzMxLjkyMiwwWiIgZmlsbD0iIzc4YmUyMCIvPjxwYXRoIGQ9Ik0xOTEuOTM3LDM4NEg2OC4wNzhBNTIuMTM3LDUyLjEzNywwLDAsMSwxNiwzMzEuOTIyVjY4LjA3OEE1Mi4xMzcsNTIuMTM3LDAsMCwxLDY4LjA3OCwxNkgxNzIuNjQzbDE2LTE2SDY4LjA3OEE2OC4xNTYsNjguMTU2LDAsMCwwLDAsNjguMDc4VjMzMS45MjJBNjguMTU2LDY4LjE1NiwwLDAsMCw2OC4wNzgsNDAwSDE3NS45MzdaIiBmaWxsPSIjMDBjMWQ1Ii8+PHBhdGggZD0iTTMyMC43NjMsMzUyLjY0NEgxNDcuNTQ5YTMxLjQ3NiwzMS40NzYsMCwwLDEtMzEuNDQxLTMxLjQ0VjE0My42NzNhMzEuNDc2LDMxLjQ3NiwwLDAsMSwzMS40NDEtMzEuNDQxSDMyMC43NjNhMzEuNDc2LDMxLjQ3NiwwLDAsMSwzMS40NCwzMS40NDFWMzIxLjJBMzEuNDc2LDMxLjQ3NiwwLDAsMSwzMjAuNzYzLDM1Mi42NDRaTTE0Ny41NDksMTI4LjIzMmExNS40NTksMTUuNDU5LDAsMCwwLTE1LjQ0MSwxNS40NDFWMzIxLjJhMTUuNDU4LDE1LjQ1OCwwLDAsMCwxNS40NDEsMTUuNDRIMzIwLjc2M0ExNS40NTcsMTUuNDU3LDAsMCwwLDMzNi4yLDMyMS4yVjE0My42NzNhMTUuNDU4LDE1LjQ1OCwwLDAsMC0xNS40NC0xNS40NDFabS00Ni45MzMsMTQzLjAxSDc4LjI4NEExNS40NTgsMTUuNDU4LDAsMCwxLDYyLjg0NCwyNTUuOFY3OC4yN2ExNS40NTcsMTUuNDU3LDAsMCwxLDE1LjQ0LTE1LjQ0SDI1MS41YTE1LjQ1NywxNS40NTcsMCwwLDEsMTUuNDQsMTUuNDRWOTYuMzI0aDE2Vjc4LjI3QTMxLjQ3NiwzMS40NzYsMCwwLDAsMjUxLjUsNDYuODNINzguMjg0YTMxLjQ3NiwzMS40NzYsMCwwLDAtMzEuNDQsMzEuNDRWMjU1LjhhMzEuNDc2LDMxLjQ3NiwwLDAsMCwzMS40NCwzMS40NDFoMjIuMzMyWiIgZmlsbD0iIzAwOTFkYSIvPjwvc3ZnPg==
-      mediatype: image/svg+xml
+  description: 'Vsphere Kubernetes Drivers Operator helps to install/manage vsphere drivers required to run kubernetes workload on vSphere.'
```

Comment:
can we add the rest of the description as well.

Reply:
the problem is with lint not taking anything above 180 characters in 1 line, will check if we can add multiple lines